### DISCUSSION TITLE
Compression by extension, how?

### DISCUSSION THREAD

**Discussion by mikae1** [borgbackup/borg#7882] [Q&A] [answered]
@ThomasWaldmann [says](https://github.com/borgbackup/borg/issues/1372#issuecomment-234841983) in [this thread](https://github.com/borgbackup/borg/issues/1372):
> borg 1.1 (after release or current master branch) has configurable compression (by extension + also by autodetect).

How can I use the “compression by extension” feature? Can not find it in `borg help compression`.

**Comment by ThomasWaldmann** [comment, marked as answer]
See the tickets linked in the last comment.

TL;DR: we decided otherwise, just use heuristics ("auto" compression) instead of hard to maintain list of extensions.

**Comment by mikae1** [comment]
> TL;DR: we decided otherwise, just use heuristics ("auto" compression) instead of hard to maintain list of extensions.

Thanks!
